### DISCUSSION TITLE
Real Time Communication not working

### DISCUSSION THREAD

**Discussion by razdacoder** [supabase/supabase#16443] [Questions]
I have a supabase project which i set up last month and everything works fin. I came to it yesterday and the real time communication is not working. I checked the logs of the project and saw this
`Error starting Postgres Extention: {:error,
 {:shutdown,
  {:failed_to_start_child, Extensions.PostgresCdcRls.Migrations,
   {%DBConnection.ConnectionError{
      message: "connection not available and request was dropped from queue after 15000ms. This means requests are coming in and your connection pool cannot serve them fast enough. You can address this by:\n\n  1. Ensuring your database is available and that you can connect to it\n  2. Tracking down slow queries and making sure they are running fast enough\n  3. Increasing the pool_size (although this increases resource consumption)\n  4. Allowing requests to wait longer by increasing :queue_target and :queue_interval\n\nSee DBConnection.start_link/2 for more information\n",`

and this too

`
Could not create schema migrations table. This error usually happens due to the following:

  * The database does not exist
  * The "schema_migrations" table, which Ecto uses for managing
    migrations, was defined by another library
  * There is a deadlock while migrating (such as using concurrent
    indexes with a migration_lock)

To fix the first issue, run "mix ecto.create".`

**Comment by GaryAustin1** [comment]
If this is a Supabase hosted project please contact support.

**Comment by aadelgrossi** [comment]
I am facing the same problem for a while- this exact log message keeps coming up and enabling/disabling any tables from `supabase_realtime` replication has no effect on the `realtime.subscription` table, where it keeps track of what tables it will be listening for postgres changes.

<img width="1167" alt="image" src="https://github.com/supabase/supabase/assets/6358947/8710ea47-2fc0-4635-9137-75db20d2b891">


This doesn't happen on a second database I have for testing/staging with an identical schema. Contacted support a few days ago but still haven't discovered the origin nor a solution to the problem. They suggested it might have something to do with Prisma, that it could have somehow deleted or messed up the `realtime` schema

``` 
Yeah looks like something deleted your schemas. Usually this is Prisma.

You can fix it by create a Realtime schema:

create schema if not exists realtime;
````

But running this SQL statement did nothing, the schema and its tables `schema_migrations` / `subscription` are already there. When using prisma on one of the clients, I never used its migrations to send local changes from my `schema.prisma` to supabase, only used it to sync with changes done through the Supabase Admin panel with `prisma db pull`

@razdacoder did you manage to fix this issue for your project? Have you reached support and if so what was the response?

**Reply by GaryAustin1** [reply]
There was this comment from a dev on this ecto issue from the Supabase Discord.
![image](https://github.com/supabase/supabase/assets/54564956/0cc1fd53-9e30-485f-964b-e59ff135fc8d)
